### PR TITLE
Authenticator component translations support for Portuguese language

### PR DESCRIPTION
**Pull request by AbhishekDesai99**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by wlee221 — COMMENTED** [review]
Hello @AbhishekDesai99, thanks for your contribution! Can you add an entry about Portuguese support in our [docs](https://ui.docs.amplify.aws/components/authenticator#internationalization-i18n)?

The file is located in https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/components/authenticator/index.page.mdx. You would add `pt` to Line 376-382 lists supported languages in alphabetical order.

**Comment by AbhishekDesai99** [comment]
> Hello @AbhishekDesai99, thanks for your contribution! Can you add an entry about Portuguese support in our [docs](https://ui.docs.amplify.aws/components/authenticator#internationalization-i18n)?
> 
> The file is located in https://github.com/aws-amplify/amplify-ui/blob/main/docs/src/pages/components/authenticator/index.page.mdx. You would add `pt` to Line 376-382, which lists supported languages in alphabetical order.

Hello @wlee221, docs are updated with the new language support.

File: packages/ui/src/i18n/dictionaries/authenticator/pt.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  'Hide password': 'Esconder a senha',
+  Loading: 'Carregando',
+  Username: 'Nome do usuário',
+  'New password': 'Nova Senha',
+  Password: 'Senha',
+  'Phone Number': 'Número de telefone',
+  'Resend Code': 'Reenviar código',
+  'Reset your password': 'Redefina sua senha',
+  'Send Code': 'Enviar código',
+  'Send code': 'Enviar código',
```
Comment (wlee221) [inline packages/ui/src/i18n/dictionaries/authenticator/pt.ts:31]: Duplicate entries here:
```suggestion
  'Send Code': 'Enviar código',
```

File: packages/ui/src/i18n/dictionaries/authenticator/pt.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  Sending: 'Enviando',
+  'Setup TOTP': 'Configurar TOTP',
+  'Show password': 'Mostrar senha',
+  'Sign in': 'Entrar',
+  'Sign In': 'Entrar',
```
Comment (wlee221) [inline packages/ui/src/i18n/dictionaries/authenticator/pt.ts:36]: Duplicate entries here as well
```suggestion
  'Sign in': 'Entrar',
```

**Review by wlee221 — APPROVED** [review]
Looks good, thanks @AbhishekDesai99! Just made a minor commit to remove duplicates. Thanks for your contribution!

**Pull request merged**